tts/Object.def.lua

```diff
@@ -358,9 +358,6 @@ function Object.setPosition(position) end
 ---@return true
 function Object.setPositionSmooth(position, collide, fast) end
 
---- Spawns snap points from a list of parameters.
----@param parameters tts__Object_SnapPoint[]
-function Object.setSnapPoints(parameters) end
```

Comment:
Could you please restore the alphabetical order?

For working code I understand that sorting things alphabetically can be a bit of a fool's game, as refactoring breaks the whole thing. However, seems as these files are purely documentation, I think there's value to the consistency.

Grouping functions by their relation to each other certainly has merit (and is how I tend to write code). However, it is subjective, and since we're not author's of the API in question we don't really have the authority to say which methods are most related to which other methods.

Reply:
For the record there are other functions not in alphabetical order, so it was not clear that that was the spec.  Done

Reply:
Understood.

This file _ought_ to be alphabetical, however other files definitely aren't. Particularly `Vector` and `Color` as their types were built by literally copy and paste from https://github.com/Berserk-Games/Tabletop-Simulator-Lua-Classes/